src/main/java/no/nav/pto/veilarbportefolje/arenapakafka/ytelser/YtelsesRepositoryV2.java

```diff
@@ -15,17 +15,9 @@
 
 import static java.util.stream.Collectors.toList;
 import static no.nav.pto.veilarbportefolje.arenapakafka.ArenaUtils.getLocalDateTimeOrNull;
-import static no.nav.pto.veilarbportefolje.database.PostgresTable.YTELSESVEDTAK.AKTORID;
-import static no.nav.pto.veilarbportefolje.database.PostgresTable.YTELSESVEDTAK.ANTALLPERMITTERINGSUKER;
-import static no.nav.pto.veilarbportefolje.database.PostgresTable.YTELSESVEDTAK.ANTALLUKERIGJEN;
-import static no.nav.pto.veilarbportefolje.database.PostgresTable.YTELSESVEDTAK.ANTALLUKERIGJENUNNTAK;
-import static no.nav.pto.veilarbportefolje.database.PostgresTable.YTELSESVEDTAK.PERSONID;
-import static no.nav.pto.veilarbportefolje.database.PostgresTable.YTELSESVEDTAK.RETTIGHETSTYPEKODE;
-import static no.nav.pto.veilarbportefolje.database.PostgresTable.YTELSESVEDTAK.SAKSID;
-import static no.nav.pto.veilarbportefolje.database.PostgresTable.YTELSESVEDTAK.SAKSTYPEKODE;
-import static no.nav.pto.veilarbportefolje.database.PostgresTable.YTELSESVEDTAK.STARTDATO;
-import static no.nav.pto.veilarbportefolje.database.PostgresTable.YTELSESVEDTAK.UTLOPSDATO;
-import static no.nav.pto.veilarbportefolje.database.PostgresTable.YTELSESVEDTAK.YTELSESTYPE;
+import static no.nav.pto.veilarbportefolje.database.PostgresTable.OpensearchData.ANTALLDAGERIGJEN;
```

Comment:
Oh, isn't this also supposed to come from YTELSESVEDTAK?

Reply:
aha, true!